bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -82,6 +81,14 @@ def __init__(self, module, is_ipython):
         # Setup import hook
         self._set_import_hook()
 
+    def AddressOf(self, *args):
+        # Return a bytearray that can fit a long long, which is what addressof
+        # returns (wrapped in a Python integer). The bytes of the bytearray
+        # correspond to the address of the object in args
+        import struct
+        ad = self.addressof(*args)
+        return bytearray(struct.pack('q', ad))
```

Comment:
Just a small detail, what is the endianess here? I suppose there is a sensible default, though I woudl check.

Reply:
https://docs.python.org/3/library/struct.html

Reply:
Thanks for the heads up, the documentation says:
```
By default, C types are represented in the machine’s native format
and byte order, and properly aligned by skipping pad bytes if necessary
(according to the rules used by the C compiler).

So when storing the long long with the address it will just pick the machine's default :smile: 
```